### ISSUE TITLE
[Feature request] Menu Buttons and more... 

### ISSUE THREAD

**Issue by Ircas** [streamaserver/streama#847]
After adding a lot of movies/tvshows the dashboard seems a bit outdated and difficult to manage.

My idea is to add 2-3 more menu buttons next to browse. A movies button and a tvshows button.

Upon pressing those buttons you will get redirected to a new page that will show only movies or tvshows. 

Also to make the main dashboard more modern. Get rid of the ajax load more button and just link it to above mentioned pages. Maybe show only 2-3 rows of videos. 

Also if this ever makes it to an update add the ajax load more button only on those pages not dashboard. 

Wanna make it even better? 
Make it so that when the user gets to the end of the page to auto load the next bunch of data (like it works on fb and Instagram).
And for the icing on the cake add something in the admin section to enable disable all options or change the number of videos shown on the dashboard. 

I know @dularion is going to kill me for adding so many things on 1 post but its all relevant...

Gimme your likes dislikes and comments please...

**Comment by dularion** [comment]
> I know @dularion is going to kill me for adding so many things on 1 post but its all relevant...

haha :D  

Love your suggestions! only problem with the auto-load-more is, that you them dominantly see one type of media, either movie or shows, and to see tv-shows you would have to scroll, scroll, scroll until all movies are auto-loaded and only then tv-shows would show up... 
Or we mix the media types? 
or we do it like I programmed it in the app, with never-ending right-scrolling carousels.

**Comment by Ircas** [comment]
Hm... I think I was misunderstood... 
Let me rephrase and try to explain.

Have the dashboard as is but without the load more ajax button just link it to the new pages and show only 2 3 rows of each. 

Create a new page called movies add there the auto load more ONLY showing movies. 

Create a new page called tvshows there the auto load more ONLY showing tv shows. 

Maybe create a new page for other videos. 
(I dont know if this is a feuture that is used by many. I perosnaly do not use it. Maybe get rid of it?) 

So I imagine the following setup. 

The home page / dash that can show the recommended and 2 3 rows of each. 

The movies page that will be quite similar to Dash. Maybe called dash.movies. This will show only movies. Maybe add the recommended slider too. And maybe change search properties to only search for movies. 

The tvshows page that will be quite similar to Dash. Maybe called dash.tvshows. This will show only tv shows. Maybe add the recommended slider too. And maybe change search properties to only search for tv shows. 

In my imagination this should be quite easy. Duplicate the dash get rid of sections not needed and rename it add a link to it and its done. 

Of course it could need thousands of lines of code and lot of hours. You should know better. 

If you still cannot understand the logic behind this I could gladly make some mockup photos

**Comment by dularion** [comment]
No no I got all that, I was simply referring to this part:
> Wanna make it even better?
> Make it so that when the user gets to the end of the page to auto load the next bunch of data (like it works on fb and Instagram).

But if that is only on those specific sub-dashboards then it makes sense again

**Comment by Ircas** [comment]
Yes that was the idea to have it only on the Sub Dash on a seperate page... 
So how likely is this to be added?

Not that I am in a rush or pressure but I hate to get excited for this and then the post gets burried 🤤
